example2/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ set(EXAMPLE2_SRCS
   ${CMAKE_CURRENT_SOURCE_DIR}/example2.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/timer.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/timer.h
+  ${CMAKE_CURRENT_SOURCE_DIR}/../inc/crnlib.h
```

Comment:
Headers of a dependent library don't really belong in the source list. Same goes for example3

Reply:
Isn't it what CMake use to know the source has to be rebuilt if the header changes?

Reply:
No. AFAIK the only use is to index it in IDE projects.

Reply:
Ah, wouldn't it be a good idea though? I see there are some VS projects there but they are likely unmaintained and will be out of date once this is merged.

Reply:
No. In the VS project setting, headers that are part of the Crunch library should be in the library's sources, not the example sources.

Reply:
OK I removed them.